### PR TITLE
Update version pins

### PR DESCRIPTION
**Pull request by ahuang11**
Update pins; use pyarrow, make preprocess more robust.

Nebari changes at https://github.com/holoviz/holoviz/pull/407

### CODE REVIEW

File: examples/data/preprocessing/earthquake_data.py [hunk 1]
```diff
@@ -24,23 +26,18 @@
...
 
+df = pd.read_parquet(os.path.join(THIS_DIR, '..', 'earthquakes.parq'))
 
-# import numpy as np
-# import pandas as pd
-# from holoviews.util.transform import lon_lat_to_easting_northing
+cleaned_df = df.copy()
+cleaned_df['mag'] = df.mag.where(df.mag > 0)
+cleaned_df = cleaned_df.set_index("time")
+cleaned_df.index = pd.to_datetime(cleaned_df.index)
```
Comment (ahuang11) [inline examples/data/preprocessing/earthquake_data.py:38]: Some notebooks set the time as index, some do not.

Will fix notebooks in a separate PR.

Comment (ahuang11) [inline examples/data/preprocessing/earthquake_data.py]:
```suggestion
cleaned_df['time'] = pd.to_datetime(cleaned_df['time'])
```

File: examples/data/preprocessing/earthquake_data.py
Comment (maximlt) [inline examples/data/preprocessing/earthquake_data.py:1]: Did you update the file stored at https://datashader-data.s3.amazonaws.com/earthquakes-projected.parq? In fact, an improvement would be to store the data files on datasets.holoviz.org.

Comment (ahuang11) [reply examples/data/preprocessing/earthquake_data.py:1]: It should produce the same result; just fixed the pathing.

How do you upload to datasets.holoviz.org?

Comment (maximlt) [reply examples/data/preprocessing/earthquake_data.py:1]: > It should produce the same result; just fixed the pathing.

I've seen differences between fastparquet and pyarrow.

File: examples/tutorial/00_Setup.ipynb [hunk 2]
```diff
@@ -111,6 +111,125 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": []
+  },
+  {
+   "cell_type": "code",
```
Comment (maximlt) [inline examples/tutorial/00_Setup.ipynb]: Empty cells.

File: examples/tutorial/01_Overview.ipynb [hunk 3]
```diff
@@ -157,6 +157,90 @@
     "\n",
     "This functionality builds on the native APIs for all the other libraries discussed above (plus many others in the SciPy/PyData ecosystem!), but in many cases hvPlot `.plot()`, hvPlot `.interactive()`, and a bit of Panel are all you need to learn to get your work done!"
    ]
+  },
+  {
+   "cell_type": "code",
```
Comment (maximlt) [inline examples/tutorial/01_Overview.ipynb]: Empty cells.

File: examples/tutorial/02_Plotting.ipynb [hunk 4]
```diff
@@ -732,6 +731,76 @@
     "\n",
     "As you can see, hvPlot makes it simple to explore your data interactively, with commands based on the widely used Pandas `.plot()` API but now supporting many more features and different types of data. The visualizations above just touch the surface of what is available from hvPlot, and you can explore the [hvPlot website](https://hvplot.holoviz.org) to see much more, or just explore it yourself using tab completion (`df.hvplot.`_[TAB]_). The following section will focus on how to put these plots together once you have them, linking them to understand and show their structure."
    ]
+  },
+  {
+   "cell_type": "code",
```
Comment (maximlt) [inline examples/tutorial/02_Plotting.ipynb]: Empty cells.

**Review by maximlt — APPROVED** [review]
@ahuang11 feel free to merge after you've cleaned up the empty cells that show up in a few files.

**Pull request merged**